benches/benches/vm_initialization.rs

```diff
@@ -110,56 +100,14 @@ pub fn vm_initialization(c: &mut Criterion) {
         group.throughput(Throughput::Bytes(tx_size as u64));
         group.bench_function(name, |b| {
             b.iter(|| {
-                unoptimized_vm_initialization_with_allocating_full_range_of_memory(
-                    &mut vm, &tx,
+                #[allow(clippy::unit_arg)]
+                black_box(
```

Comment:
Does this `black_box` actually do anything?

Reply:
Removing `black_box` improves speed 
<img width="735" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/3b32f05e-a668-4314-bacd-5e53f5c23207">